<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMeasure</class>
 <widget class="QMainWindow" name="frmMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measure distance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Point1</string>
    </property>
    <widget class="QLabel" name="lblThe_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint1_x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint1_y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint1_z">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxStart_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Point2</string>
    </property>
    <widget class="QLabel" name="lblThe_start_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint2_x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint2_y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPoint2_z">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxStart_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Distance</string>
    </property>
    <widget class="QLabel" name="lblThe_start_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDistance_x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDistance_y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDistance_z">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDistance_total">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblThe_start_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>total</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
